pkg/kv/kvserver/replica_evaluate.go

```diff
@@ -506,8 +506,15 @@ func evaluateCommand(
 			}
 			return s
 		}
+		var resp string
+		switch r := reply.(type) {
```

Comment:
Maybe something that generalizes better is

```go
if reply.Size() > 1024 /* 1kb */ {
  resp = trunc(r.ResponseHeader.String())
} else {
  resp = trunc(reply.String())
}
```

Reply:
Good point, done.